src/lib/codemirror-editor.ts

```diff
@@ -67,6 +67,8 @@ export class CodeMirrorEditorElement extends LitElement {
   // that we can set this value internally without triggering an update.
   private _value?: string;
 
+  private _capturedCodeMirrorStyles: HTMLStyleElement[] = [];
```

Comment:
initialize to `undefined` to skip the lit-html array handling when adoptedStyleSheets is available

Reply:
Done.